src/db.js

```diff
@@ -204,82 +193,107 @@
 
                 return {
                     desc,
+                    distinct,
                     execute,
                     filter,
-                    distinct,
+                    limit,
                     map
                 };
             };
-            const filter = function (...args) {
-                filters.push(args.slice(0, 2));
+
+            const limit = function (...args) {
```

Comment:
why would you use a rest parameter here, wouldn't it be simpler to just define `start` and `end`? Seems like ES6 for the sake of ES6